packages/blade/src/components/Accordion/__tests__/Accordion.web.test.tsx

```diff
@@ -0,0 +1,196 @@
+import userEvent from '@testing-library/user-event';
+import { useState } from 'react';
+import { waitFor } from '@testing-library/react';
+import { AccordionItem } from '../AccordionItem';
+import { Accordion } from '../Accordion';
+import renderWithTheme from '~src/_helpers/testing/renderWithTheme.web';
+import assertAccessible from '~src/_helpers/testing/assertAccessible.web';
+import { Button } from '~components/Button';
+
+describe('<Accordion />', () => {
+  it('should render', () => {
+    const { container } = renderWithTheme(
+      <Accordion>
```

Comment:
question: do we need to expose styled props on root Accordion for layout purposes?

Reply:
Right, root Accordion has a top level `BaseBox` which accepts these

Reply:
So Accordion will also have styled props, right? We will have to pass it down to the BaseBox?